docs/BREAKINGCHANGES.md

```diff
@@ -0,0 +1,217 @@
+Enable shebang use of PowerShell on non-Windows platforms. This means on Unix based systems,
+you can make a script executable which would invoke PowerShell automatically rather than explicitly invoking pwsh.
+This also means that you can now do things like `powershell foo.ps1` or `powershell fooScript` without specifying `-File`.
+However, this change now requires that you explicitly specify `-c` or `-Command` when trying to do things like `powershell.exe Get-Command`.
+
+### Implement Unicode escape parsing [#3958](https://github.com/PowerShell/PowerShell/issues/3958)
+TODO
+
+
+### Change New-ModuleManifest encoding to UTF8NoBOM on non-Windows platforms [#3940](https://github.com/PowerShell/PowerShell/issues/3940)
+Previously, `New-ModuleManifest` creates psd1 manifests in UTF-16 with BOM, creating a problem for Linux tools.  This breaking change changes the encoding of `New-ModuleManifest` to be UTF (no BOM) in non-Windows platforms.
+
+
+### Prevent Get-ChildItem from recursing into symlinks (#1875). [#3780](https://github.com/PowerShell/PowerShell/issues/3780)
+This change brings `Get-ChildItem` more in line with the Unix `ls -r` and the Windows `dir /s` native commands.  Like the mentioned commands, the cmdlet will display simbolic links to directories found during recursion, but will not recurse into them.
+
+
+### Fix `Get-Content -Delimiter` to not include the delimiter in the returned lines [#3706](https://github.com/PowerShell/PowerShell/issues/3706)
+Previously, the output while using `Get-Content -Delimiter` was inconsistent and inconvient as it required further processing of the data to remove the delimiter.
+This change removes the delimiter in returned lines.
+
+
+### Implement Format-Hex in C# [#3320](https://github.com/PowerShell/PowerShell/issues/3320)
+ The `-raw` parameter is now No-op. Going forward all of the output will be displayed with a true representation of numbers that includes all of the bytes for its type (what the Raw parameter was formally doing).
+
+
+### PowerShell as a default shell doesn't work with script command [#3319](https://github.com/PowerShell/PowerShell/issues/3319)
+On Unix, it is a convention for shells to accept `-i` for an interactive shell and many tools expect this behavior (`script` for example, and when setting powershell as the default shell) and calls the shell with the `-i` switch. This change is breaking in that -i previously could be used as short hand to match `-inputformat` which now will need to be `-in`.
+
+
+### Completions for environment variables differ between platforms [#3227](https://github.com/PowerShell/PowerShell/issues/3227)
+The casing of `$PSModulePath` has been corrected to be consistent with Windows PowerShell.
+TODO: not sure if this actually a breaking change.  If the change is to make something consistent with Windows, there would be no difference between 6.0 and 5.1
```

Comment:
I think you can remove this, this was only a breaking change between releases of PSCore6